polaris-react/src/components/DropZone/components/FileUpload/FileUpload.tsx

```diff
@@ -71,6 +75,18 @@ export function FileUpload(props: FileUploadProps) {
       break;
     case 'small':
       viewMarkup = <img width="20" src={uploadArrow} alt="" />;
+      if (polarisSummerEditions2023) {
```

Comment:
Unsure why this was an img in the first place, however we need this to be an icon so it can inherit fill colour styles based on state.